.github/workflows/eval-lib-tests.yml

```
@@ -0,0 +1,27 @@
+name: "Building Nixpkgs lib-tests"
+
+permissions:
+  contents: read
+
+on:
+  pull_request_target:
+    branches-ignore:
+      - 'staging'
```

Does this support globs? We have staging-next and all the staging branches for nixos relases:

```suggestion
      - 'staging-*'
```

---

Yep, we need to use:

```suggestion
      - 'staging'
      - 'staging-**'
```

Though, I guess we should let it build on `staging-next` branches, since it is supposed to be a bugfix branch for hydra.

---

If we had the fix that @infinisil suggests, we don't need this part.